### ISSUE TITLE
BUG: During setup Initialize command fails, throws sqlalchemy.exc.InvalidRequestError

### ISSUE THREAD

**Issue by jtrip** [FlaskCon/traveller#11]
Tested running Python 3.8.12 and 3.9.2.
Mysql version 8.0.23
macOS 11.4

Basic setup, clone repo, create and activate venv, `pip install -r requirements.txt`

Add mysql connection string to instance the config file `traveller/instance/config.py`:
```SQLALCHEMY_DATABASE_URI = 'mysql+pymysql://root:x@localhost/traveller'```

then run:
`python manage.py initialize`

that command fails, throws an exception:
`
sqlalchemy.exc.InvalidRequestError: When initializing mapper mapped class ReviewerList->reviewer_lists, expression 'User' failed to locate a name ('User'). If this is a class name, consider adding this relationship() to the <class 'modules.conf.models.ReviewerList'> class after both dependent classes have been defined.
`

**Comment by Abdur-rahmaanJ** [comment]
For me it's ok

Win 8.1
Python 3.8.6
MySQL 5.7.35

...

@rehmanis

**Comment by rehmanis** [comment]
@Abdur-rahmaanJ 
same error for me

```
Ubuntu 20.04
MySQL 8.0.19
python 3.8.10
```

`pymysql` works fine when using https://github.com/shopyo/shopyo-factory and then manually install `pymysql` and `crypography`

**Comment by jugmac00** [comment]
Same error for me, also Ubuntu, but 18.04 - and additionally I tried to run it with SQLite.

Interestingly, I also experience errors when trying to run `pytest`:

```
❯ pytest
======================================================= test session starts ========================================================
platform linux -- Python 3.8.11, pytest-6.2.5, py-1.10.0, pluggy-1.0.0
rootdir: /home/jugmac00/Projects/traveller
collecting ... [Errno 2] No such file or directory: 'config_demo.json'
collected 0 items / 1 error                                                                                                        

============================================================== ERRORS ==============================================================
__________________________________________________ ERROR collecting test session ___________________________________________________
venv/lib/python3.8/site-packages/sqlalchemy/ext/declarative/clsregistry.py:322: in _resolve_name
    rval = d[token]
venv/lib/python3.8/site-packages/sqlalchemy/util/_collections.py:729: in __missing__
    self[key] = val = self.creator(key)
venv/lib/python3.8/site-packages/sqlalchemy/ext/declarative/clsregistry.py:301: in _access_cls
    return self.fallback[key]
E   KeyError: 'User'

The above exception was the direct cause of the following exception:
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:170: in exec_module
    exec(co, module.__dict__)
traveller/conftest.py:12: in <module>
    from app import create_app
traveller/app.py:235: in <module>
    app = create_app(environment)
traveller/app.py:177: in create_app
    mod = importlib.import_module("modules.{}.view".format(folder))
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
traveller/modules/conf/view.py:3: in <module>
    from modules.conf.forms import ConfForm
traveller/modules/conf/forms.py:5: in <module>
    class ConfForm(ModelForm):
venv/lib/python3.8/site-packages/wtforms_alchemy/__init__.py:116: in __init__
    generator.create_form(cls)
venv/lib/python3.8/site-packages/wtforms_alchemy/generator.py:141: in create_form
    for key, property_ in sa.inspect(self.model_class).attrs.items():
venv/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py:893: in __get__
    obj.__dict__[self.__name__] = result = self.fget(obj)
venv/lib/python3.8/site-packages/sqlalchemy/orm/mapper.py:2383: in attrs
    configure_mappers()
venv/lib/python3.8/site-packages/sqlalchemy/orm/mapper.py:3299: in configure_mappers
    mapper._post_configure_properties()
venv/lib/python3.8/site-packages/sqlalchemy/orm/mapper.py:1965: in _post_configure_properties
    prop.init()
venv/lib/python3.8/site-packages/sqlalchemy/orm/interfaces.py:197: in init
    self.do_init()
venv/lib/python3.8/site-packages/sqlalchemy/orm/relationships.py:2077: in do_init
    self._process_dependent_arguments()
venv/lib/python3.8/site-packages/sqlalchemy/orm/relationships.py:2148: in _process_dependent_arguments
    self.target = self.entity.persist_selectable
venv/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py:893: in __get__
    obj.__dict__[self.__name__] = result = self.fget(obj)
venv/lib/python3.8/site-packages/sqlalchemy/orm/relationships.py:2044: in entity
    argument = self.argument()
venv/lib/python3.8/site-packages/sqlalchemy/ext/declarative/clsregistry.py:326: in _resolve_name
    self._raise_for_name(name, err)
venv/lib/python3.8/site-packages/sqlalchemy/ext/declarative/clsregistry.py:304: in _raise_for_name
    util.raise_(
venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py:182: in raise_
    raise exception
E   sqlalchemy.exc.InvalidRequestError: When initializing mapper mapped class ReviewerList->reviewer_lists, expression 'User' failed to locate a name ('User'). If this is a class name, consider adding this relationship() to the <class 'modules.conf.models.ReviewerList'> class after both dependent classes have been defined.
===================================================== short test summary info ======================================================
ERROR  - sqlalchemy.exc.InvalidRequestError: When initializing mapper mapped class ReviewerList->reviewer_lists, expression 'User...
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
========================================================= 1 error in 0.83s =========================================================
```

I'd appreciate any help, as this looks like a major blocker as now at least 3 volunteers cannot work on the code base.

**Comment by jtrip** [comment]
Another volunteer found a solution and discussed in discord, posting here for anyone interested while they build a PR.

in: `traveller/modules/conf/models.py`
add: `from modules.box__default.auth.models import User`

this resolves the issue for me on 3.8.12, I will be testing on 3.9.2 as well.
